conda/common/io.py

```diff
@@ -323,6 +323,8 @@ def stderr_log_level(level, logger_name=None):
 def attach_stderr_handler(
     level=WARN, logger_name=None, propagate=False, formatter=None
 ):
+    from ..gateways.logging import TokenURLFilter
```

Comment:
This unfortunately breaks a API promise we've made in https://github.com/conda/conda/blob/7fa76c9665a07687cdea028efa4024a8fc33847d/conda/common/__init__.py#L6-L7

Reply:
😬 Let me look into it, maybe we can pass it as they are doing with `formatter` too.

Reply:
You can take another look now @jezdez